code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/witch.dm

```diff
@@ -47,14 +45,63 @@
 						/obj/item/recipe_book/magic = 1,
 						/obj/item/chalk = 1
 						)
+
+	var/classes = list("Old Magick", "Godsblood", "Mystagogue")
+	var/classchoice = input("How do your powers manifest?", "THE OLD WAYS") as anything in classes
+
+	var/shapeshifts = list("Zad", "Cat", "Cat (Black)", "Bat", "Lesser Volf")
+	var/shapeshiftchoice = input("What form does your second skin take?", "THE OLD WAYS") as anything in shapeshifts
+
+	switch (classchoice)
+		if("Old Magick")
+			// the original witch: arcyne t2 (buffed from t1) with 6 spellpoints
+			ADD_TRAIT(H, TRAIT_ARCYNE_T2, TRAIT_GENERIC)
+			H.adjust_skillrank(/datum/skill/magic/arcane, 1, TRUE)
+			H.mind?.adjust_spellpoints(9) // twelve if you pick arcyne potential
+			beltl = /obj/item/storage/magebag/associate
+		if("Godsblood")
+			//miracle witch: capped at t2 miracles. cannot pray to regain devo, but has high innate regen because of it (2 instead of 1 from major)
+			var/datum/devotion/D = new /datum/devotion/(H, H.patron)
+			H.adjust_skillrank(/datum/skill/magic/holy, 1, TRUE)
+			D.grant_miracles(H, cleric_tier = CLERIC_T2, passive_gain = CLERIC_REGEN_WITCH, devotion_limit = CLERIC_REQ_2)
+			D.max_devotion *= 0.5
+			neck = /obj/item/clothing/neck/roguetown/psicross/wood
+		if("Mystagogue")
+			// hybrid arcane/holy witch with t1 arcane and t1 miracles, but less spellpoints, lower max devotion and less regen (0.5). Still can't pray.
+			var/datum/devotion/D = new /datum/devotion/(H, H.patron)
+			H.adjust_skillrank(/datum/skill/magic/holy, 1, TRUE)
+			D.grant_miracles(H, cleric_tier = CLERIC_T1, passive_gain = CLERIC_REGEN_MINOR, devotion_limit = CLERIC_REQ_1)
+			D.max_devotion *= 0.5
+			ADD_TRAIT(H, TRAIT_ARCYNE_T1, TRAIT_GENERIC)
+			H.adjust_skillrank(/datum/skill/magic/arcane, 1, TRUE)
+			H.mind?.adjust_spellpoints(6) // twelve if you pick arcyne potential
+			beltl = /obj/item/storage/magebag/associate
+			neck = /obj/item/clothing/neck/roguetown/psicross/wood
+
 	if(H.mind)
-		H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/guidance)
-		H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/projectile/arcynebolt)
-		H.mind.AddSpell(new /obj/effect/proc_holder/spell/self/message)
+		switch (shapeshiftchoice)
+			if("Zad")
+				H.mind.AddSpell(new /obj/effect/proc_holder/spell/targeted/shapeshift/crow/witch)
+			if("Cat")
```

Comment:
Perhaps just keep it with black cats, as they fit thematically with witches, instead of pink ones? Open to discussion on this one.